packages/babel-plugin/src/utils/label.js

```diff
@@ -137,7 +148,7 @@ function getDeclaratorName(path, t) {
   }
 
   let variableDeclarator = parent.findParent(p => p.isVariableDeclarator())
-  if (!variableDeclarator) {
+  if (!variableDeclarator || !variableDeclarator.get('id').isIdentifier()) {
```

Comment:
@kddc this was the more appropriate fix. The `isArrowFunctionExpression` block that you have introduced has covered the introduced test case but I believe that it was more about improving label extraction for this specific case than about fixing a crash (the latter was a byproduct of the former though, in that scenario).

By patching the underlying issue here I've managed to also fixed the weird test case introduced as `no-label-obj-pattern-computed-property`

Reply:
@Andarist Awesome - that's even better then :)